framework/rbac/migrations/m170907_052038_rbac_add_index_on_auth_assignment_user_id.php

```diff
@@ -10,7 +10,7 @@
 class m170907_052038_rbac_add_index_on_auth_assignment_user_id extends Migration
 {
     public $column = 'user_id';
-    public $index = 'idx-auth_assignment-user_id';
```

Comment:
I'd rather use `idx-auth_assignment-user_id` because `idx_user_id` is too general and naming conflicts are likely to occur.

Reply:
Then index name should be adjusted also in `.sql` files.

Reply:
Yes.